src/main/java/frc/robot/commands/Drive.java

```diff
@@ -40,15 +40,15 @@ public Drive(DrivebaseSubsystem drivebase, DoubleSupplier translateX, DoubleSupp
   @Override
   public void execute() {
     // Reversed strafe and rotation axis
-    double x = getDeadzone(-translateX.getAsDouble()) * getSpeedMultiplier();
-    double y = getDeadzone(translateY.getAsDouble()) * getSpeedMultiplier();
-    double z = getDeadzone(-rotateZ.getAsDouble());
+    double x = getDeadzone(-translateX.getAsDouble()) * getSpeedMultiplier() * Dashboard.maxSpeedEntry.get().getDouble();
+    double y = getDeadzone(translateY.getAsDouble()) * getSpeedMultiplier() * Dashboard.maxSpeedEntry.get().getDouble();
```

Comment:
You already use `Dashboard.maxSpeedEntry.get().getDouble()` in `getSpeedMultiplier()`, so you are applying the speed multiplier twice.

Reply:
I just realized that.